migrations/init_migration.up.sql

```diff
@@ -30,7 +31,8 @@ CREATE TABLE IF NOT EXISTS community (
   tagsCount SMALLINT DEFAULT 0,
   followingUsers INT DEFAULT 0,
   ipfsHash VARCHAR(66),
-  ipfsHash2 VARCHAR(66)
+  ipfsHash2 VARCHAR(66),
+  network VARCHAR(20)
```

Comment:
Network needs to be `networkId` of type  INT. It is enough to store just the network id.

There should be another table, `Networks` with fields id and name. Then add foreign key from `networkId` column to `network` table.